aesara/tensor/basic_opt.py

```diff
@@ -2990,7 +2995,7 @@ def constant_folding(fgraph, node):
 
         # TODO: `Type` itself should provide an interface for constructing
         # instances appropriate for a given constant.
-        if isinstance(output.type, TensorType):
+        if isinstance(output.type, DenseTensorType):
```

Comment:
I changed TensorType to DenseTensorType here, because otherwise it trying to create dense constant with sparse data inside.